### ISSUE TITLE
Inconsistency of lore/dialogue

### ISSUE THREAD

**Issue by BlackLordRoo** [Archival1/CV-BugReporting#19]
**Describe the bug**
Not bug, just minor inconsistency. One of respawn (recumbobulate) dialogues said that clothes can't be recovered. But doberman guards after respawn have their radio and stun gun!

**Expected behavior**
Guards could to lost their equipment until retrieved new one from an armory. 

**Desktop (please complete the following information):**
 - OS: Windows
 - Graphics Card AMD
 - Version cef91e5c

**Comment by without-ordinary** [comment]
I think its any clothing or accessories that aren't stripped when naked, like Captain Roo's hat and the Rat Janitor's mop/brush.

**Comment by naelstrof** [comment]
Yeah, unfortunately it was by necessity as the CV police just have inherent abilities to shoot their gun and call other cops. Would be cool to add some fidelity to the characters and equipment so that they can lose it.
